src/Controls/samples/Controls.Sample/Pages/Controls/IndicatorPage.xaml

```diff
@@ -0,0 +1,128 @@
+                <RowDefinition Height="50"/>
+                <RowDefinition Height="*"/>
+            </Grid.RowDefinitions>
+            <Label
+                Text="Basic"
+                Style="{StaticResource Headline}"/>
+            <IndicatorView Grid.Column="1"
+                           Count="5"
+                           Position="2"
+                           HorizontalOptions="Center" />
+            <Label
+                Grid.Row="1"
+                Text="Colors"
+                Style="{StaticResource Headline}"/>
+            <IndicatorView Grid.Row="1" GridLayout.Column="1" Count="5" Position="2"  HorizontalOptions="Center" SelectedIndicatorColor="Red" IndicatorColor="Blue" Background="Yellow" />
+            <Label
+                Grid.Row="2"
+                Text="Indicator Shape"
+                Style="{StaticResource Headline}"/>
+            <IndicatorView Grid.Row="2" GridLayout.Column="1" Count="5" Position="2"  HorizontalOptions="Center"  IndicatorsShape="Square"/>
+            <Label
+                Grid.Row="3"
+                Text="Indicator Size"
+                Style="{StaticResource Headline}"/>
+            <IndicatorView  Grid.Row="3" GridLayout.Column="1" Count="5" Position="2"  HorizontalOptions="Center"  IndicatorSize="15"/>
+            <Label
+                Grid.Row="4"
+                Text="Indicator HideSingle"
+                Style="{StaticResource Headline}"/>
+            <IndicatorView Grid.Row="4" GridLayout.Column="1" Count="1" HorizontalOptions="Center" HideSingle="True"/>
+            <Label
+                Grid.Row="5"
+                Text="Indicator MaximumVisible -  7 of 10"
+                Style="{StaticResource Headline}"/>
+            <IndicatorView Grid.Row="5" GridLayout.Column="1" Count="10" Position="3" MaximumVisible="7" HorizontalOptions="Center"/>
+            <Label
+                Grid.Row="6"
+                Text="Indicator Template"
+                Style="{StaticResource Headline}"/>
+            <IndicatorView Grid.Row="6" GridLayout.Column="1"
+                   Count="5"
+                   Position="2"
+                   IndicatorColor="Transparent"
+                   SelectedIndicatorColor="Transparent"
+                   HorizontalOptions="Center">
+                <IndicatorView.IndicatorTemplate>
+                    <DataTemplate>
+                        <Label Text="&#xf30c;" FontFamily="Ionicons" Style="{StaticResource IndicatorLabelStyle}" />
+                    </DataTemplate>
+                </IndicatorView.IndicatorTemplate>
+            </IndicatorView>
+            <Label
+                Grid.Row="7" 
+                Text="Using with CarouselView"
+                Style="{StaticResource Headline}" VerticalOptions="Start" />
+            <StackLayout Grid.Row="7"  Grid.Column="1"  VerticalOptions="Start" >
+                <CarouselView IndicatorView="indicatorView" VerticalOptions="Start" HeightRequest="100" Loop="False">
```

Comment:
CarouselVIew doesn't crashes iff Loop =true on Android, and doesn't show when Loop=False